AntWars/Board/Ants/Ant.cs

```diff
@@ -34,8 +34,13 @@ public abstract class Ant : BoardObject
         internal IAIAnt AI { get; set; }
         internal Board board;
 
-        internal Ant(Board board, Player owner)
+        internal Ant(Board board, Player owner, int viewRange, int maxInventory)
         {
+            // TODO berechnen!
+            Cost = 1;
```

Comment:
Eigentlich unsinnig die Kosten in der Ameise zuspeichern, da man sie nur 1x somit das bitte entfernen